Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -0,0 +1,204 @@
+            return window;
+        }
+
+        [MenuItem("Amazon GameLift/Show Amazon GameLift Window", priority = 0)]
+        public static void ShowWindow()
+        {
+            GetWindow();
+        }
+
+        [MenuItem("Amazon GameLift/Bring Panel to Front", priority = 1)]
+        public static void FocusPanel()
+        {
+            ShowWindow();
+        }
+
+        [MenuItem("Amazon GameLift/Set AWS Account Profiles", priority = 100)]
+        public static void OpenAccountProfilesTab()
+        {
+            GetWindow().OpenTab("Credentials");
+        }
+
+        [MenuItem("Amazon GameLift/Host with Anywhere", priority = 101)]
+        public static void OpenAnywhereTab()
+        {
+            GetWindow().OpenTab("Anywhere");
+        }
+
+        [MenuItem("Amazon GameLift/Host with Managed EC2", priority = 102)]
+        public static void OpenEC2Tab()
+        {
+            GetWindow().OpenTab("EC2");
+        }
+
+        [MenuItem("Amazon GameLift/Import Sample Game", priority = 103)]
+        public static void ImportSampleGame()
+        {
+            string filePackagePath = $"Packages/{Paths.PackageName}/{Paths.SampleGameInPackage}";
+            AssetDatabase.ImportPackage(filePackagePath, interactive: true);
+        }
+
+        [MenuItem("Amazon GameLift/Help/Documentation", priority = 200)]
+        public static void OpenDocumentation()
+        {
+            Application.OpenURL(Urls.AwsHelpGameLiftUnity);
+        }
+
+        [MenuItem("Amazon GameLift/Help/AWS GameTech Forum", priority = 201)]
+        public static void OpenGameTechForums()
+        {
+            Application.OpenURL(Urls.AwsGameTechForums);
+        }
+
+        [MenuItem("Amazon GameLift/Help/Report Issues", priority = 202)]
+        public static void OpenReportIssues()
+        {
+            Application.OpenURL(Urls.GitHubAwsLabs);
+        }
+
+        private void CreateGUI()
+        {
+            _root = rootVisualElement;
+            _mVisualTreeAsset = UnityEngine.Resources.Load<VisualTreeAsset>("EditorWindow/GameLiftPlugin");
+            if (_mVisualTreeAsset == null)
+            {
+                return;
+            }
+
+            VisualElement uxml = _mVisualTreeAsset.Instantiate();
+            _root.Add(uxml);
+            ApplyText();
+
+            CoreApi = CoreApi.SharedInstance;
+            var awsCredentials = AwsCredentialsFactory.Create();
+            CreationModel = awsCredentials.Creation;
+            UpdateModel = awsCredentials.Update;
+            RefreshProfiles();
+            
+            var contentContainer = _root.Q(className: "main__content").Children().First();
+            var landingPage = new LandingPage(contentContainer);
+            _userProfilesPage = new AwsUserProfilesPage(contentContainer,this);
+
+            _tabButtons = _root.Query<Button>(className: TabButtonClassName).ToList();
+            _tabContent = _root.Query(className: TabContentClassName).ToList();
```

Comment:
what is the difference b/w _root.Q and _root.Query? From my understanding both of them are meant to do the same thing? If so, can you please be consistent everywhere and use Query?

Reply:
Q is used to reference a single element (first if multiple which we don't utilize) within the UI, whereas Query returns everything that the query matches. So for single elements like a Label with a unique name, we would use Q. But for Query we would use it to get all of 1 element with a class name, then for example clear all the text fields.